tests/console/apache.pm

```diff
@@ -135,7 +135,7 @@ sub run {
 
             # Run and test this new environment
             assert_script_run 'httpd2-prefork -f /tmp/prefork/httpd.conf';
-            assert_script_run 'until ps aux|grep wwwrun; do echo waiting for httpd2-prefork pid; done';
+            assert_script_run 'until ps aux|grep wwwrun|grep httpd2-prefork; do echo waiting for httpd2-prefork pid; done';
```

Comment:
Just looking on my openQA instance, grep of wwwrun or httpd2-prefork is the same or not ?
```
wwwrun   24267  0.0  0.0 110672  5232 ?        S    00:41   0:00 /usr/sbin/httpd-prefork -DSYSCONFIG -DSSL -C PidFile /run/httpd.pid -C Include /etc/apache2/sysconfig.d//loadmodule.conf -C Include /etc/apache2/sysconfig.d//global.conf -f /etc/apache2/httpd.conf -c Include /etc/apache2/sysconfig.d//include.conf -DSYSTEMD -DFOREGROUND -k start
wwwrun   26874  0.0  0.0 110672  5232 ?        S    02:25   0:00 /usr/sbin/httpd-prefork -DSYSCONFIG -DSSL -C PidFile /run/httpd.pid -C Include /etc/apache2/sysconfig.d//loadmodule.conf -C Include /etc/apache2/sysconfig.d//global.conf -f /etc/apache2/httpd.conf -c Include /etc/apache2/sysconfig.d//include.conf -DSYSTEMD -DFOREGROUND -k start
wwwrun   26884  0.0  0.0 110672  5204 ?        S    02:25   0:00 /usr/sbin/httpd-prefork -DSYSCONFIG -DSSL -C PidFile /run/httpd.pid -C Include /etc/apache2/sysconfig.d//loadmodule.conf -C Include /etc/apache2/sysconfig.d//global.conf -f /etc/apache2/httpd.conf -c Include /etc/apache2/sysconfig.d//include.conf -DSYSTEMD -DFOREGROUND -k start
wwwrun   26909  0.0  0.0 110672  5204 ?        S    02:26   0:00 /usr/sbin/httpd-prefork -DSYSCONFIG -DSSL -C PidFile /run/httpd.pid -C Include /etc/apache2/sysconfig.d//loadmodule.conf -C Include /etc/apache2/sysconfig.d//global.conf -f /etc/apache2/httpd.conf -c Include /etc/apache2/sysconfig.d//include.conf -DSYSTEMD -DFOREGROUND -k start
wwwrun   29390  0.0  0.0 110672  5204 ?        S    04:16   0:00 /usr/sbin/httpd-prefork -DSYSCONFIG -DSSL -C PidFile /run/httpd.pid -C Include /etc/apache2/sysconfig.d//loadmodule.conf -C Include /etc/apache2/sysconfig.d//global.conf -f /etc/apache2/httpd.conf -c Include /etc/apache2/sysconfig.d//include.conf -DSYSTEMD -DFOREGROUND -k start
```

Reply:
Another thing is `httpd2-prefork` or `httpd-prefork`, on my leap15.4 it's just `httpd-prefork`

Reply:
Good comments,
Since the 'pid' is forked via `httpd2-prefork` cmd line, then I would prefer to grep both.

If we only grep one word, it will return 0 even no daemon is there.

It is `httpd2-prefork`, my codes are up to date :)

Reply:
Yes, but it can change, regex `grep -E "httpd(2)?-pre"` would be safer ?

Reply:
Why Not, let me do, thanks much!

VR is on the way

Reply:
https://openqa.opensuse.org/tests/3052434
https://openqa.suse.de/tests/10345962
https://openqa.opensuse.org/tests/3052433